lib/resolvers/http.js

```diff
@@ -58,9 +95,16 @@ module.exports = {
       u.protocol = url.parse(location.href).protocol;
     }
 
-    return fetch(this.makeRequest(url.format(u)))
-      .then((response) => response.arrayBuffer())
-      .then((buffer) => Buffer.from(buffer))
+    const request = new Request(url.format(u), {
+      method: "GET",
+      headers: this.headers,
+      redirect: this.redirects,
```

Comment:
I suggest we drop it.
[`redirect`](https://developer.mozilla.org/en-US/docs/Web/API/Request/redirect) is an entirely different setting in case of `Request`.

Reply:
```suggestion
```